test/models/vrp_test.rb

```diff
@@ -277,5 +277,25 @@ def test_available_interval
 
       assert_equal [5, 6, 7, 9], TestHelper.create(vrp).services.first.unavailable_days.sort
     end
+
+    def test_timewindow_start_can_not_be_nil
+      vrp = VRP.basic
+      vrp[:services].first[:activity][:timewindows] = [{ end: 10 }]
+      created_vrp = TestHelper.create(vrp)
+      assert_equal 0, created_vrp.services.first.activity.timewindows.first.start
+
+      vrp = VRP.basic
+      vrp[:services].first[:activities] = [{ timewindows: [{ start: nil, end: 10 }] },
+                                           { timewindows: [{ start: nil, end: 10 }] }]
```

Comment:
maybe like this, to cover all possibilities
```suggestion
      vrp[:services].first[:activities] = [{ timewindows: [{ start: nil, end: 10 }] },
                                           { timewindows: [{ end: 10 }] }]
```